documentation/source/users/rmg/input.rst

```diff
@@ -230,8 +230,64 @@ The following is an example of a typical species item, based on methane using SM
 			"""
 	)
 
-.. _reactionsystem:
+.. _forbidden_structures:
+
+Forbidden Structures
+=====================
+
+RMG exlores a wide varitey of structures as a builds a mechanism.
+Sometimes, RMG makes nonsensical, unphysical, or unstable species that make their way into the core.
+
+Luckily, RMG has a remedy to prevent these undesirable structures from making their way into your mechanism.
```

Comment:
What luck! 🤣

Reply:
This all sounds a bit "chatty" to me. But if that's the style of the rest of the manual, it's fine. Not a big problem anyway - at least we have documentation now!